bundles/net.tourbook/src/net/tourbook/data/TourData.java

```diff
@@ -2435,7 +2436,7 @@ private void computeAltimeterGradientSerieWithFixedInterval() {
          final float distanceDiff = distanceSerie[indexHigh] - distanceSerie[indexLow];
          final float altitudeDiff = altitudeSerie[indexHigh] - altitudeSerie[indexLow];
 
-         final float timeDiff = deviceTimeInterval * (indexHigh - indexLow);
+         final int timeDiff = deviceTimeInterval * (indexHigh - indexLow);
 
          // keep altimeter data
          dataSerieAltimeter[serieIndex] = 3600 * altitudeDiff / timeDiff / UI.UNIT_VALUE_ELEVATION;
```

Comment:
No because in the end, `UI.UNIT_VALUE_ELEVATION` is a float. Also, all those variables are `int` (`deviceTimeInterval * (indexHigh - indexLow)`) so i dont think that declaring them as a float would change anything.
Additionally, I think the unit tests would have caught an eventual loss of acuracy

Reply:
The question is, how is `.../timeDiff/...` internally in Java converted.

I did a simple test with `int` and `float` and both results are the same